src/04-modality-specific-files/03-electroencephalography.md

```diff
@@ -304,6 +304,10 @@ REF   -0.0742   -0.0200  -0.0100   cup       Ag/AgCl
 GND   0.0742    -0.0200  -0.0100   cup       Ag/AgCl
 ```
 
+The `acq` parameter can be used to indicate acquisition of the same data (i.e.,
+the electrode positions) with, for example, a different electrode position
+recording device.
```

Comment:
I propose to add the before/after example like this

... different electrode position recording device, or repeated digitization before and after the recording.